### ISSUE TITLE
Subject in custom badge forces lowercase

### ISSUE THREAD

**Issue by garetmckinley** [badges/shields#2112]
Not sure if this is a bug or a stylistic choice, but I noticed that custom badges only respect the case of the status, not the subject.

Example: `https://img.shields.io/badge/TEST-CASE-green.svg` ![](https://img.shields.io/badge/TEST-CASE-green.svg)

I swear I've used badges with uppercase in the subject line, but maybe I'm remembering incorrectly. I first noticed the issue on http://makeapullrequest.com/ the URL Kent uses has `PRs` in it, but the badge looks odd rendering at `prs`. Which makes me think it's a bug that was introduced over time

**Comment by mabbas64** [comment]
The same has happend to all my badges. They were nicely formated like  "MyTitle | 1234"  and now they're all lowercased during last week or two somehow.  and read "mytitle | 1234" 

e.g.: 

```
<img src="https://img.shields.io/badge/EQ BankCard 8 - 735 | 1-6b8b08.svg">
```

**Comment by chris48s** [comment]
Hi thanks for reporting this. It looks like this was introduced in #2058 although it is not the intended behaviour. I've proposed a fix in #2116
